### DISCUSSION TITLE
agent_config.yaml

### DISCUSSION THREAD

**Discussion by ncd1992** [open-telemetry/opentelemetry-collector-contrib#8828] [Q&A]
Hello is it possible to use a base64 encoded password instead of clear test in the config file under receivers ?

**Comment by jpkrohling** [comment]
Let's keep the discussion on the core repo: https://github.com/open-telemetry/opentelemetry-collector/discussions/5077

**Reply by pmm-sumo** [reply]
Since the question here is a duplicate of the above (and the answer was provided there), I think we could remove this one for clarity? WDYT @jpkrohling?
